.gitlab/software_composition_analysis.yml

```diff
@@ -0,0 +1,26 @@
+---
+# software_composition_analysis stage
+# Contains jobs to generate SBOMs for the repository,
+# upload them to Datadog, and scan them for vulnerabilities.
+
+datadog-sca-ci:
+  stage: software_composition_analysis
+  tags:
+    - arch:amd64
+    - specific:true
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci-sbom:2024011006
+  when: always
+  allow_failure: true
+  dependencies: []
+  script:
+    # disabling tracing to avoid leaking secrets
+    # see https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin:
+    # "Using ‘+’ rather than ‘-’ causes these options to be turned off"
+    - set +o xtrace
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name "ci.datadog-agent.datadog_api_key_org2" --with-decryption --query "Parameter.Value" --out text)
+    - export DD_APP_KEY=$(aws ssm get-parameter --region us-east-1 --name "ci.datadog-agent.datadog_app_key_org2" --with-decryption --query "Parameter.Value" --out text)
+    - export DATADOG_APP_KEY=$DD_APP_KEY
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why are we exporting the App key variable "twice?"